core/src/main/java/org/elasticsearch/index/mapper/core/KeywordFieldMapper.java

```diff
@@ -128,6 +120,8 @@ public KeywordFieldMapper build(BuilderContext context) {
                 } else if (propName.equals("ignore_above")) {
                     builder.ignoreAbove(XContentMapValues.nodeIntegerValue(propNode, -1));
                     iterator.remove();
+                } else if (TypeParsers.parseNorms(builder, propName, propNode, parserContext)) {
```

Comment:
I don't like that this uses the same lenient version the legacy stuff uses...can this just be straightfoward here, looking for norms as a boolean and not using this parseNorms method?